### PR TITLE
fix a typo

### PR DESCRIPTION
**Pull request by yorkerlin**
@iglesias

### CODE REVIEW

**Comment by iglesias** [comment]
Would be nice if you can also fix indentation starting in line 100. The first level of indentation of the method has two spaces, which does not correspond to the first level of the other methods in the same file.

File: src/shogun/optimization/L1Penalty.h [hunk 1]
```diff
@@ -112,12 +112,10 @@ class L1Penalty: public SparsePenalty
 	  if (CMath::abs(variable)<m_rounding_eplison)
 		  variable=0.0;
 	  return variable;
-		return 0;
 	}
```
Comment (iglesias) [inline src/shogun/optimization/L1Penalty.h]: Leave a newline also here if so.

**Comment by iglesias** [comment]
Merging. Travis failures not caused by this change.

**Comment by karlnapf** [comment]
Hi guys,

such patches should be labelled properly. Rather than "fix a typo", it should be "add forgotten return 0 statement"

It is impossible to sort things later otherwise, or track down bugs from commit logs in the buildbot

**Comment by iglesias** [comment]
You are definitely right. Thanks for the heads up!

**Pull request merged**